src/components/footer/footer.scss

```diff
@@ -15,21 +15,29 @@
   }
 }
 
+.App-footer {
+  background-color: #00478f;
+}
+
 .footer-content {

```

Comment:
We should nest the lists under another element. That element should have `space-between` together with the right part of the footer. The lists could have a `min-width: 250px; margin-left: 30px` for all the lists except the first one. This way the space between the columns is not that large. Same as in Figma 🙏